src/Services/OpenexhangeratesManager.php

```diff
@@ -0,0 +1,159 @@
+<?php
+
+namespace Lab19\Cart\Services;
+
+use Illuminate\Support\Facades\Cache;
+
+class OpenexhangeratesManager
```

Comment:
@AlexeiaW I'm wondering if this service is agnostic of the type of exchange rate api used, in which case we could just rename this  ExchangeRatesManager instead, rather than a specific reference to the api. What do you think?

Reply:
That makes sense to me, keep it more generic. 

I renamed the class.